package/generateScss.mjs

```diff
@@ -62,6 +63,65 @@ const generateSCSSModule = async (moduleName, importObj) => {
 			generatedScss += `${key}: ${value};\n`;
 		});
 
+	// animations.scss
+  	} else if (moduleName.toLowerCase() === 'animations') {
+  	generatedScss = "@use 'easings' as _e;\n@use 'media' as _mq;\n@use 'sass:string';\n$id: string.unique-id();\n\n";
+
+  	const createAnimationMixin = (animationName, keyframesContent, duration, easing) => {
+    		return `@mixin ${animationName} {
+      			$name: op-#{$id}-${animationName}; ${keyframesContent}
+      			animation: #{$name} ${duration} ${easing};
+    		}\n`;
+  	};
+
+  	const fadeInBloomPart = Animations['--animation-fade-in-bloom'].split(' ');
+  	const darkNameFIB = fadeInBloomPart[0].replace('fade-in-bloom', 'fade-in-bloom-dark');
+  	const darkDurationFIB = fadeInBloomPart[1];
+  	const darkEasingFIB = fadeInBloomPart[2].replace(/var\(--(.*?)\)/g, '#{_e.$$$1}');
+  	const fadeOutBloomPart = Animations['--animation-fade-out-bloom'].split(' ');
+  	const darkNameFOB = fadeOutBloomPart[0].replace('fade-out-bloom', 'fade-out-bloom-dark');
+  	const darkDurationFOB = fadeOutBloomPart[1];
+  	const darkEasingFOB = fadeOutBloomPart[2].replace(/var\(--(.*?)\)/g, '#{_e.$$$1}');
+  	const keyframeFIBDark = Animations['--animation-fade-in-bloom-@media:dark'].replace(/@keyframes\s+(\S+)/, '@keyframes #{$name}');
+  	const keyframeFOBDark = Animations['--animation-fade-out-bloom-@media:dark'].replace(/@keyframes\s+(\S+)/, '@keyframes #{$name}');
```

Comment:
is it possible to avoid some of this hardcoded duplication and make it more programmatic? that way any future `@media:dark` queries can also be handled